packages/cli-kit/src/colors.ts

```diff
@@ -0,0 +1,4 @@
+import {createRequire} from 'module'
+
+const require = createRequire(import.meta.url)
+export const colors = require('ansi-colors')
```

Comment:
this is now the best way to import commonjs dependencies

Reply:
I'd add a comment there explaining that we have to do it with `ansi-colors` because it only supports CJS